graphics/cairo-devel/Portfile

```diff
@@ -51,12 +52,6 @@ depends_lib                 path:lib/pkgconfig/pixman-1.pc:libpixman \
 
 minimum_xcodeversions       {8 2.4.1}
 
-# Prevent cairo from using librsvg, libspectre, poppler.
-patchfiles-append           patch-configure.diff
-
-# Don't check stderr in configure tests.
-patchfiles-append           dont-check-stderr.patch
```

Comment:
I've left a question regarding this patch here: https://gitlab.freedesktop.org/cairo/cairo/-/issues/121